docs/sources/administration/plugin-management/plugin-install.md

```diff
@@ -21,11 +21,24 @@ weight: 120
 
 # Install a plugin
 
-Besides the UI, you can use alternative methods to install a plugin depending on your environment or set-up.
+{{< admonition type="caution" >}}
+
+Plugin installation from the Grafana website will be removed in February 2026. Use the options in this document instead.
+
+{{< /admonition >}}
+
+The most common way to install a plugin is through the Grafana UI.
```

Comment:
Maybe this needs a subheader to allow easier linking to it 🤷 i'm still struggling a bit with the overlapping topics on the section home vs sub pages